docs/RemovingPresets.md

```diff
@@ -0,0 +1,91 @@
+[Readme](README.md) » Removing Presets
+
+## Removing Presets
+
+1. [Introduction](#introduction)
+2. [FSXA-UI](#fsxa-ui)
+3. [TailwindCSS](#tailwindcss)
+4. [Vue-TSX-Support](#vue-tsx-support)
+5. [Semantic-Release](#semantic-release)
+
+### Introduction
+
+This chapter introduces optional dependencies. Why we have them in our project and how you can easily remove them.
+### FSXA-UI
```

Comment:
The introduction currently grammatically really only is one sentence (either change '.' to ',', followed by 'why', or ajust the second sentence.